### PR TITLE
doc: Onboarding documentation

### PR DESCRIPTION
**Pull request by e-martinsson**
## Describe your changes

Fixes: 
#839 by adding documentation for creating a super user for accessing a locally hosted AIS instance
#840 by adding documentation for creating a new fair in a locally hosted AIS instance.

## Checklist before requesting review

- [X] Feature/fix PRs should add one atomic (as small as possible) feature or fix.
- [x] The code compiles and all the tests pass.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -46,6 +46,42 @@ The web-server will setup everything and connect itself to a postgis database, a
...
+
+### Creating a super user account
+
+You will notice that there is no way to login to the system. This is because there are no users in the newly created, empty database. To solve this, you have to enter the Docker container and create a super user, by following the steps below.
+
+1. First list the Docker processes using the command ```docker ps```. Find the process for the ais-web Docker container and note down the value in field "CONTAINER ID".
+
+2. Then, you want to execute an interactive ```sh``` shell on the ais-web container, with the following command:
+
+    ```docker exec -it containerid sh```
```
Comment (Hampfh) [inline docs/README.md:61]: (Just some info don't have to fix): When doing block-selections that shouldn't take up the entire line, just using a single ´ around the text will suffice.
Tripple backticks are only required when you want results like this:
image

**Review by Hampfh — APPROVED** [review]
Thanks Elias! Nice work! 👍

**Comment by Hampfh** [comment]
Also don't forget to add your reviewer (or reviewers) in GitHub. You can do this here
image

**Review by didrikmunther — APPROVED** [review]
LGTM! Good job!

**Pull request merged**